Wrappers.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <iostream>
+#include "CoffeeShop.h"
+#include "InvoiceManager.h"
+#include "ScooterRental.h"
+#include "Studio.h"
+#include "Town.h"
+
```

Comment:
Don't include what you don't use!
You aren't using any of these types in this header file.
Forward declare pointer types and reference types when they aren't used.
include the header files where they are used: in the cpp files
- [ ] revise